src/_scss/components/checkbox/_filterCheckbox.scss

```diff
@@ -37,11 +37,17 @@
         display: flex;
         align-items: flex-start;
 
+        &:last-of-type {
+          margin-bottom: 0;
+        }
+
         input {
           @include usa-input-checkbox;
           margin: 0;
           border: solid rem(1) $gray-cool-20;
-          min-width: rem(14);
```

Comment:
this min-width is still needed, but it should be 12 now if the height and width are set to 12. You can see one of the checkboxes is the wrong width now, i think because it has a long title that wraps

Reply:
Added back in the min-width!